main/src/com/google/refine/util/HttpClient.java

```diff
@@ -131,6 +137,28 @@ public void process(
                     }
                 });
 
+        if (System.getProperty("http.proxyHost") != null) {
+            proxyHost = System.getProperty("http.proxyHost");
+        }
+
+        if (System.getProperty("https.proxyHost") != null) {
+            proxyHost = System.getProperty("https.proxyHost");
```

Comment:
Let us remove this section since HTTPS proxies are not supported.

Reply:
Sure